### ISSUE TITLE
Password change option

### ISSUE THREAD

**Issue by DenisD3D** [TheShark34/S-Update-Server#16]
You can change your password in the settings panel. You can valid with nothing in the fields and when we login the next time it's dosen't works.

**Comment by Litarvan** [comment]
There is a Change button on bottom : https://github.com/Litarvan/S-Update-Server/blob/7160b9ef2e0369fa1ab2455a8abdd05c66e634a7/views/settings.twig#L61

**Comment by DenisD3D** [comment]
Yes, but when you leave the password field empty and you click change, your password change to "". The login interface need some input in the field for login so we can't login. I have delete my config folder. Now it's works.

**Comment by Litarvan** [comment]
Oh i agree this is not really normal, i'll work on a fix

**Comment by DenisD3D** [comment]
Ok, thanks
